x-pack/plugins/security_solution/common/detection_engine/schemas/response/rules_schema.mocks.ts

```diff
@@ -24,45 +23,69 @@ export const getRulesSchemaMock = (anchorDate: string = ANCHOR_DATE): RulesSchem
   from: 'now-6m',
   immutable: false,
   name: 'Query with a rule id',
-  query: 'user.name: root or user.name: admin',
   references: ['test 1', 'test 2'],
-  severity: 'high',
+  severity: 'high' as const,
   severity_mapping: [],
   updated_by: 'elastic_kibana',
   tags: ['some fake tag 1', 'some fake tag 2'],
   to: 'now',
-  type: 'query',
   threat: [],
   version: 1,
   output_index: '.siem-signals-default',
   max_signals: 100,
   risk_score: 55,
   risk_score_mapping: [],
-  language: 'kuery',
   rule_id: 'query-rule-id',
   interval: '5m',
   exceptions_list: getListArrayMock(),
   related_integrations: [],
   required_fields: [],
   setup: '',
+  throttle: 'no_actions',
+  actions: [],
+  building_block_type: undefined,
+  note: undefined,
+  license: undefined,
+  outcome: undefined,
+  alias_target_id: undefined,
+  alias_purpose: undefined,
+  timeline_id: undefined,
+  timeline_title: undefined,
+  meta: undefined,
+  rule_name_override: undefined,
+  timestamp_override: undefined,
+  timestamp_override_fallback_disabled: undefined,
+  namespace: undefined,
 });
 
-export const getRulesMlSchemaMock = (anchorDate: string = ANCHOR_DATE): RulesSchema => {
-  const basePayload = getRulesSchemaMock(anchorDate);
-  const { filters, index, query, language, ...rest } = basePayload;
+export const getRulesSchemaMock = (anchorDate: string = ANCHOR_DATE): FullResponseSchema => ({
+  ...getResponseBaseParams(anchorDate),
+  query: 'user.name: root or user.name: admin',
+  type: 'query',
+  language: 'kuery',
+  index: undefined,
+  data_view_id: undefined,
+  filters: undefined,
+  saved_id: undefined,
+});
 
+export const getRulesMlSchemaMock = (anchorDate: string = ANCHOR_DATE): FullResponseSchema => {
   return {
-    ...rest,
+    ...getResponseBaseParams(anchorDate),
     type: 'machine_learning',
     anomaly_threshold: 59,
     machine_learning_job_id: 'some_machine_learning_job_id',
   };
 };
 
-export const getThreatMatchingSchemaMock = (anchorDate: string = ANCHOR_DATE): RulesSchema => {
+export const getThreatMatchingSchemaMock = (
+  anchorDate: string = ANCHOR_DATE
+): FullResponseSchema => {
```

Comment:
Any reason why `getThreatMatchingSchemaMock` couldn't return type of `ThreatMatchResponseSchema`? 
As well as the above `getRulesMlSchemaMock` have a retrun type of `MachineLearningResponseSchema`?

Reply:
Nope, good point - updated to use more specific return types throughout this file.